account_global_discount/views/report_account_invoice.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[@id='total']/div/table//tr[1]" position="before">
```

Comment:
With this position in report, this information only shows when invoice already paid, in addtion, it shows after total amout. You should change it by: `//div[@id='total']/div/table/t[@t-set='tax_totals'] position="before"`

Reply:
thanks